### ISSUE TITLE
illegal server response on APPEND

### ISSUE THREAD

**Issue by vzaliva** [qnikst/HaskellNet#83]
`APPEND` command gives an error on Gmail server response. Here is the session log:
```
HaskellNet-SSL SEND: "000002 APPEND CMU   {3152}"
HaskellNet-SSL SEND: "\r\n"
HaskellNet-SSL RECV: "+ go ahead\r"
user error (illegal server response)
```

Other commands (`LOGIN`, `LIST`) worked fine.

Version info:

HaskellNet 0.5.1

**Comment by vzaliva** [comment]
It might have been fixed in a newer version but in order to test I need to find a working combination of `HaskellNet`, `HaskellNet-SSL`, and `network`  packages. So far I've failed. The main issue is:

    error: HaskellNet-SSL >   Module ‘Network.Socket.Internal’ does not export ‘PortNumber’

This is an incompatible API change that was introduced around version 2.8.0 of the Network package. I just could not find a combination of dependencies version to install HaskellNet-0.6 via stack
